### ISSUE TITLE
Insufficient Color Contrast

### ISSUE THREAD

**Issue by inghamn** [City-of-Bloomington/drupal-theme-cob#278]
WCAG requires a minimum color contrast ratio of 4.5:1. People with vision disabilities such as limited vision and color blindness may have difficulty differentiating the blue headings and links from the white background.

### Headings and Links
The color contrast ratio of the blue (#3f9dff) text (headings and links) on a white (#ffffff) background is 2.81:1. 
vars.css:
```scss
$color-heading: $dodger-blue;
```

### Related Links
The color contrast ratio of the blue (#3f9dff) text (headings and links) on a white (#ffffff) background is 2.81:1.
vars-fn1.scss
```scss
$color-action: $fern
```

### Attachments
In the "Attachments" section, the link text and surrounding text is the same color, and styling has been removed from the links making it very difficult for sighted users to know that the items are links.  These are examples of plain links styled in html-elements.

### Search Form
The placeholder gray is too washed out, both in the header form and the homepage form.  These are text-color that's been lightened.

The green button is not a high enough contrast with the gray.  It is styled in buttons.scss.

### Info Cards
The hover background color ($color-selected-background: $havelock-blue)does not have enough contrast with the card text (#fff).  It might be worth changing the $color-selected-background to a darker color.

**Comment by inghamn** [comment]
The approach so far was to keep the background colors (ie. light blue and green) and alter the foreground color on elements, such as buttons.

This change is a bit too dramatic, and folks want to keep the light text on background color buttons and elements.  We should go back to the white text on colors and alter the background colors to be dark enough to contrast with white.

This affects the light blue and green (dodger-blue, mayor-sushi)
